language-support/ts/daml-ledger/BUILD.bazel

```diff
@@ -40,7 +39,7 @@ eslint_test(
     name = "lint",
     srcs = glob(
         ["**/*.ts"],
-        exclude = ["lib/**/*"],
+        exclude = ["lib/**/*", "node_modules/**/*"]
```

Comment:
I'm a bit worried that I have to exlude the 'node_modules' directory here. Otherwise eslint decends into it in the runfiles.

Reply:
The alternative is to list all files explicitly like we do for the `da_ts_library` above. Maybe it's worth to split this out in a `filegroup`. Let's to that in a separate PR though. Here and for `@daml/react` as well.

Reply:
I personally feel that `glob` in build systems is an anti-pattern. I would be very happy if we explicitly enumerated the modules that go into a target not just here but in general.

Reply:
ok, let's fix this in a followup PR.